README.rst

```diff
@@ -1,4 +1,122 @@
 thoth-adviser
 -------------
 
-A recommendation engine for project Thoth.
+A recommendation engine for project `Thoth <https://github.com/thoth-station/>`_.
+
+There are three main goals of thoth-adviser (as of now):
+
+1. Provide a library that coverts basic operations in Python ecosystem (such as operations with package index sources, project specific operations on libraries used).
+2. Provide a tool that can compute recommendations in project `Thoth <https://github.com/thoth-station/thoth>`_.
+3. Check provenance of installed packages (which package source indexes are used - this is not guaranteed by pip nor Pipenv).
+
+To interact with a deployed Thoth, you can use the
+`Thamos CLI <https://github.com/thoth-station/tamos>`_.
+
+Provenance Checks
+=================
+
+The provenance check is done against Pipenv and Pipenv.lock that are expected
+as an input. The output is a structured report (with metadata) that states
+issues found in the application stack. There are currently reported the
+following issues:
+
+1. ``ERROR``/``ARTIFACT-DIFFERENT-SOURCE`` - reported if a package/artifact **is** installed from a different package source index in comparision to the configured one
+2. ``INFO``/``ARTIFACT-DIFFERENT-SOURCE`` - reported if a package/artifact **can be** installed from a different package source index in comparision to the configured one
```

Comment:
I would probably rename this one not to create confusion with 1.

Reply:
ja, this is confusing... isnt it `ARTIFACT-POSSIBLE-DIFFERENT-SOURCE` ?